### PR TITLE
Move marketing year to invoice line

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: test/unit/enrichment/enrich-invoice-line.test.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const mockGetSchemeCode = require('../../../app/enrichment/get-scheme-code')
 
 const scheme = require('../../mocks/scheme')
 const { SCHEME_CODE } = require('../../mocks/values/scheme-code')
+const marketingYear = 2023
```
Comment (Kirsten1822) [inline test/unit/enrichment/enrich-invoice-line.test.js:6]: I think this should be below enrichInvoiceLine, but above let invoiceLine

File: test/unit/enrichment/process-invoice-lines.test.js [hunk 2]
```diff
@@ -8,12 +8,15 @@ const { NET_DESCRIPTION, GROSS_DESCRIPTION } = require('../../mocks/values/descr
...
 const invoiceLine = require('../../mocks/payment-requests/invoice-line')
+
 const invoiceLines = [
   invoiceLine,
   invoiceLine,
   { ...invoiceLine, description: NET_DESCRIPTION }
 ]
 
+const marketingYear = 2023
+
```
Comment (Kirsten1822) [inline test/unit/enrichment/process-invoice-lines.test.js:19]: I think these should be below processInvoiceLines

File: test/unit/enrichment/enrich-invoice-line.test.js (hunk 1 above)
Comment (johnwatson484) [reply test/unit/enrichment/enrich-invoice-line.test.js]: Agreed, moved.

File: test/unit/enrichment/process-invoice-lines.test.js (hunk 2 above)
Comment (johnwatson484) [reply test/unit/enrichment/process-invoice-lines.test.js]: Also moved.

**Review by Kirsten1822 — APPROVED** [review]
Happy to approve :)

**Pull request merged**